File: 0058-length-of-last-word/0058-length-of-last-word.py
class Solution(object):
    def lengthOfLastWord(self, s):
        """
        :type s: str
        :rtype: int
        """
        # Step 1: Strip any trailing spaces from the string
        s = s.rstrip()

        # Step 2: Split the string into a list of words
        words = s.split()

        # Step 3: Return the length of the last word in the list
        if words:
            return len(words[-1])
        else:
            return 0  # In case the string is empty or contains only spaces
        

File: 0058-length-of-last-word/0058-length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        // Step 1: Trim any trailing spaces from the string
        s = s.trim();
        
        // Step 2: Split the string into an array of words
        String[] words = s.split(" ");
        
        // Step 3: Return the length of the last word in the array
        if (words.length > 0) {
            return words[words.length - 1].length();
        } else {
            return 0;  // In case the string is empty or contains only spaces
        }
    }
}